change/@fluentui-react-toolbar-d17fdf5d-7561-468e-b616-f1bbe8f690d5.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
## [🕵 fluentuiv9 Open the Visual Regressions report to inspect the affected screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=28658&commitId=8d5434a1f54ff7aae30a0ab50e74a1918cec757b&env=prod&clientType=FLUENTUI)


  <details><summary><b>Toolbar Converged</b> 3 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Toolbar Converged.Vertical Button.Toggle On.chromium.png|125| Changed|
|Toolbar Converged.Vertical Button.default.chromium.png|136| Changed|
|Toolbar Converged.Vertical Button.Button Pressed.chromium.png|136| Changed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v9"/>